packages/components/src/internal/components/entities/models.ts

```diff
@@ -512,18 +516,20 @@ export interface IEntityTypeDetails extends IEntityDetails {
  *     typeListingSchemaQuery: SchemaQuery; // The schema query used to get the listing of all of the data type instances (e.g., all the data classes) available
  */
 export interface EntityDataType {
+    allowRelativeDateFilter?: boolean;
+    allowSingleParentTypeFilter?: boolean;
     ancestorColumnName?: string;
     appUrlPrefixParts?: string[];
     containerFilter?: Query.ContainerFilter;
     deleteHelpLinkTopic: string;
-    dependencyText: Function | string ;
+    dependencyText: Function | string;
     descriptionPlural: string;
     descriptionSingular: string;
     editTypeAppUrlPrefix?: string;
     exprColumnsWithSubSelect?: string[];
     filterArray?: Filter.IFilter[];
     filterCardHeaderClass?: string;
-    getInstanceDataType?: (schemaQuery: SchemaQuery) => string;
+    getInstanceDataType?: (schemaQuery: SchemaQuery, queryDisplayName?: string) => string;
```

Comment:
Should this be 'altQueryName'?  And perhaps a comment above on when it's useful?